src/lib/update-alternatives/UI/alternative_dialog.rb

```diff
@@ -55,13 +55,17 @@ def set_handler
       selected_choice = Yast::UI.QueryWidget(:choices_table, :CurrentItem)
       log.info("User selected the alternative: #{selected_choice}")
       @alternative.choose!(selected_choice)
-      finish_dialog
+      finish_dialog(true)
     end
 
     def auto_handler
       log.info("User selected \"Set automatic mode\" button")
       @alternative.automatic_mode!
-      finish_dialog
+      finish_dialog(true)
+    end
+
+    def cancel_handler
+      finish_dialog(false)
```

Comment:
well, currently it is by default nil, which is also evaluated as false. so this method is not needed, but if you would like to have it explicitelly, it is fine for me.